src/pages/schemas/addConnectionModalBox.tsx

```diff
@@ -0,0 +1,111 @@
+import Buttons from '@/components/ComposePage/buttons/buttons';
+import { dBConnectionPostAction } from '@/redux/actions/dbConnectionAction';
+import {Input, Modal} from 'antd'
+import axios from 'axios';
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux';
+import { toast } from 'react-toastify';
+import styles from './connectionbox.module.css'
+import { showSuccessToast } from './toast';
+
+
+interface MyModalProps {
+    showConnectionBox?: boolean;
+    onCancel?: () => void;
+}
+
+function showWarningToast(message: string) {
+    toast.warn(message);
+  }
+
+const addConnectionModalBox: React.FC<MyModalProps> = ({ showConnectionBox, onCancel }) => {
+    const [name, setName] = useState('');
+    const [description, setDescription] = useState('');
+    const [connectionURL, setconnectionURL] = useState('');
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+
+    const dispatch = useDispatch();
+
+    const handleOnClickCancel = () => {
+        setName('')
+        setDescription('')
+        setconnectionURL('')
+        setUsername('')
+        setPassword('')
+        onCancel()
+    }
+
+    const handleOnClickConnect = async () => {
+
+        if (!name || !connectionURL || !username || !password) {
+            showWarningToast('Please fill in all required fields.');
+            return;
+          }
+
+          dispatch(
+            dBConnectionPostAction(name,description,connectionURL,username,password)
+          )
+
+        console.log('name: ',name)
+        console.log('description: ', description)
+        console.log('connection url: ', connectionURL)
+        console.log('username: ',username)
+        console.log('password: ', password)
+
+        setName('')
+        setDescription('')
+        setconnectionURL('')
+        setUsername('')
+        setPassword('')
+        
+        onCancel()
+        showSuccessToast("Connection Successfull")
```

Comment:
Need to check Is API Success for showing success message.

Reply:
made changes to check API success and show success/error toast accordingly.